.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,80 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug, "unverified"]
+assignees:
+  - 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened? Also tell us, what did you expect to happen? If applicable, add screenshots to help explain your problem.
+      description: Tell us what you see!
+      placeholder: 
+    validations:
+      required: true
+
+  - type: textarea
+    id: what-did-you-expected
+    attributes:
+      label: What did you expected to happened
+      description: Tell us what you expected!
+      placeholder: 
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Please provide a way to reproduce the problem or at list what you did before getting it.
+      placeholder: 
+    validations:
+      required: true
+
+  - type: dropdown
+    id: system-info
+    attributes:
+      label: System Information
+      description: What operating system you are running the extention on?
+      options:
+        - "Windows 10"
+        - "Windows 11"
+        - "MacOS"
+        - "Linux Mint"
+        - "Debian"
+        - "NixOS"
+        - "ArchLinux"
+        - "Other"
```

should probably add android here since it's also on android. additionally windows 11 and 10 are like the same thing functionally so i merged em

```suggestion
      description: What operating system you are running the extension on?
      options:
        - "Windows 10/11"
        - "Windows (other)"
        - "Android"
        - "MacOS"
        - "Linux Mint"
        - "Debian"
        - "NixOS"
        - "Arch Linux"
        - "Other"
```

---

Il y a effectivement une version iOS de Firefox. https://www.mozilla.org/fr/firefox/browsers/mobile/ios/

---

all browsers on iOS are just safari wearing different costumes
i think technically that's allowed to change in the EU but AFAIK mozilla hasn't done it
although, now that i think about it, [Orion](https://help.kagi.com/orion/browser-extensions/ios-ipados-extensions.html) does support Firefox extensions on iOS -- zero clue how well it works though

---

Oh, yeah. Apple is a monopoly, I could I forget that. Anyway, I am fixing the template.